src/writer/command_line.py

```diff
@@ -1,134 +1,64 @@
-        sys.exit(1)
-
-    if command in ("edit", "hello") and host is not None:
-        logging.warning("Writer Framework has been enabled in edit mode with a host argument\nThis is enabled for local development purposes (such as a local VM).\nDon't expose Builder to the Internet. We recommend using a SSH tunnel instead.")
-
-    if command in ("edit", "hello") and enable_remote_edit is True:
-        logging.warning("The remote edit flag is active. Builder will accept non-local requests. Please make sure the host is protected to avoid drive-by attacks.")
-
-
-def _route(
-    command: str,
-    absolute_app_path: str,
-    port: int,
-    host: Optional[str],
-    enable_remote_edit: Optional[bool],
-    enable_server_setup: Optional[bool],
-    template_name: Optional[str],
-    api_key: Optional[str] = None,
-    env: Union[List[str], None] = None
-):
-    if host is None:
-        host = "127.0.0.1"
-    if command in ("deploy"):
-        writer.deploy.deploy(absolute_app_path, api_key, env=env)
-    if command in ("undeploy"):
-        writer.deploy.undeploy(api_key)
-    if command in ("deployment-logs"):
-        writer.deploy.runtime_logs(api_key)
-    if command in ("edit"):
-        writer.serve.serve(
-            absolute_app_path, mode="edit", port=port, host=host,
-            enable_remote_edit=enable_remote_edit, enable_server_setup=enable_server_setup)
-    if command in ("run"):
-        writer.serve.serve(
-            absolute_app_path, mode="run", port=port, host=host, enable_server_setup=True)
-    elif command in ("hello"):
-        create_app("hello", template_name="hello", overwrite=True)
-        writer.serve.serve("hello", mode="edit",
-                               port=port, host=host, enable_remote_edit=enable_remote_edit,
-                               enable_server_setup=False)
-    elif command in ("create"):
-        create_app(absolute_app_path, template_name=template_name)
+   pass
+
+@click.command()
+@click.option('--host', default="127.0.0.1", help="Host to run the app on")
+@click.option('--port', default=5000, help="Port to run the app on")
+@click.argument('path', type=click.Path(exists=True, file_okay=False, dir_okay=True, resolve_path=True))
+def run(path, host, port):
+    """Run the app from PATH folder in run mode."""
+
+    writer.serve.serve(
+        path, mode="run", port=port, host=host, enable_server_setup=True)
+
+@click.command()
+@click.option('--host', default="127.0.0.1", help="Host to run the app on")
+@click.option('--port', default=5000, help="Port to run the app on")
+@click.option('--enable-remote-edit', help="Set this flag to allow non-local requests in edit mode.", is_flag=True)
+@click.option('--enable-server-setup', help="Set this flag to enable server setup hook in edit mode.", is_flag=True)
+@click.argument('path', type=click.Path(exists=True, file_okay=False, dir_okay=True, resolve_path=True))
```

Comment:
The previous error message seems more relevant, I will recommend not using click validation and doing it in code

![image](https://github.com/writer/writer-framework/assets/159559/b79a09ea-7e0b-4094-b8ca-f716fe19a670)